src/App.tsx

```diff
@@ -1,10 +1,41 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodoList';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoFooter } from './components/TodoFooter';
+import { Notification } from './components/Notification';
 
-const USER_ID = 0;
+const USER_ID = 10620;
 
 export const App: React.FC = () => {
+  const [initialTodos, setInitialTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filter, setFilter] = useState('');
+
+  const getFiltredTodos = (value: string) => {
+    switch (value) {
+      case 'completed':
```

Comment:
consider adding an enum for that so that you won't make a mistake using strings like this
```suggestion
      case FilterType.Completed:
```